components/cloud/azure/Cargo.toml

```diff
@@ -6,18 +6,18 @@ publish = false
 
 [dependencies]
 async-trait = "0.1"
-azure_core = { version = "0.1.0", git = "https://github.com/Azure/azure-sdk-for-rust"}
+azure_core = { version = "0.1.0", git = "https://github.com/Azure/azure-sdk-for-rust" }
 azure_identity = { version = "0.1.0", git = "https://github.com/Azure/azure-sdk-for-rust" }
-azure_storage = { version = "0.1.0", git = "https://github.com/Azure/azure-sdk-for-rust", default-features = false, features = ["account", "blob"] }
+azure_storage = { version = "0.1.0", git = "https://github.com/Azure/azure-sdk-for-rust", features = ["account", "blob"] }
```

Comment:
Why remove `default-features = false`?

Reply:
Must be a mismatch when using global replace.